nodejs/lancedb/indices.ts

```diff
@@ -113,6 +113,19 @@ export interface IvfPqOptions {
   sampleRate?: number;
 }
 
+/**
+ * Options to create a full text search index
+ */
+export interface FtsOptions {
+  /**
+   * Whether to build the index with positions.
+   * True by default.
+   * If set to false, the index will not store the positions of the tokens in the text,
+   * which will make the index smaller and faster to build, but will not support phrase queries.
+   */
+  withPositions?: boolean;
```

Comment:
Minor nit: can we be consistent between position (used everywhere else) and positions (used here).  Either one is fine but we should pick one and stick with it to reduce friction for users working with multiple languages.